jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -193,7 +193,7 @@ combinations.each{
   }
   // Run tests on Windows ARM64 only if explicitly requested
   if (builderLabel =~ /^win.*COMPILED_BY.*-arm64$/) {
-    if (!parameters['RUN_ARM64_TESTS'].toString().equalsIgnoreCase("true")){
+    if (!new String(parameters['RUN_ARM64_TESTS']).toString().equalsIgnoreCase("true")){
```

Comment:
If we're using the String constructor we probably don't need the `toString()` as it should already be one?

Reply:
Fair point - fixed.